sky/execution.py

```diff
@@ -530,6 +531,11 @@ def spot_launch(
             },
             output_prefix=spot.SPOT_CONTROLLER_YAML_PREFIX)
         controller_task = task_lib.Task.from_yaml(yaml_path)
+
+        controller_resources = (controller_resources or
```

Comment:
styleguide: explicitly test controller_resources is not None

Reply:
Good catch! Fixed. Thanks!